frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -97,7 +103,7 @@ public void update(AnActionEvent anActionEvent) {
 
   protected void doRebase(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch branchToRebase) {
     Project project = anActionEvent.getProject();
-    assert project != null;
+    assert project != null : "Can't get project from anActionEvent variable";
```

Comment:
While you're here, maybe checkstyle-forbid asserts that aren't followed by any `:`